### DISCUSSION TITLE
How scikit-multilearn handles the imbalanced dataset and missing labels?

### DISCUSSION THREAD

**Discussion by island255** [scikit-multilearn/scikit-multilearn#260] [Q&A] [answered]
Thank you for proposing such a practical toolkit for multi-label classification. I am a new learner and I found it easy to get started with this tool. But when I run this toolkit for some downstream tasks, some problems come to me and I wonder whether and how scikit-multilearn tries to handle them.

The first question is that the labels in the dataset are often imbalanced. I don't know if there are measures to resample the dataset or do some things to help handle this problem when I run the model.fit(). 

The second thing is that sometimes I can only generate a dataset with missing labels. I wonder if there are some means in the tool that can help me handle this issue. 

Looking forward to your reply :)

**Comment by CquptZA** [comment, marked as answer]
you can use some sampling method
